src/providers/ad/ad_common.c

```diff
@@ -1050,6 +1051,17 @@ ad_set_sdap_options(struct ad_options *ad_opts,
         goto done;
     }
 
+    /* Warn if the user is doing something silly like overriding the schema
+     * with the AD provider
+     */
+    schema = dp_opt_get_string(id_opts->basic, SDAP_SCHEMA);
+    if (schema && strcasecmp(schema, "ad") != 0) {
```

Comment:
One nitpick that I can change before pushing (if you agree) ... I'd prefer to explicitly compare schema to NULL here.
You don't have to resubmit the patch, at all. Just let me know if you agree with the change.

Reply:
sure, go ahead